Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -88,8 +169,28 @@ public static int[] InsertLast(int[] array, int number)
         /// <returns>A new array with element inserted at a given index. If an array is empty or null, returns new array with number in it.</returns>
         public static int[] InsertAt(int[] array, int number, int index)
         {
-            // ToDo: implement.
-            return array;
+
+            if (array == null  || index == 0)
+            {
+                var newArray = new int[] { 10 };
+                return newArray;
+            }
+
+            if (index < 0 || index > array.Length) return array;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4904050)